css/src/components/scroll.scss

```diff
@@ -9,3 +19,24 @@
 	overflow-y: auto;
 	-webkit-overflow-scrolling: touch;
 }
+
+.scroll-snap-container {
+	height: max-content;
+	margin-block: 1rem;
+	scroll-snap-type: x mandatory;
+	gap: 1rem;
+	scroll-behavior: smooth;
+}
+
+.scroll-snap-item {
+	scroll-snap-align: center;
+	flex-basis: 90%;
+	height: auto;
+	min-width: 80%;
+}
+
+.jump-links {
```

Comment:
This is essentially equivalent to .button, and so I think it can be removed.

Reply:
removed